src/components/VmTemplateModal/VmTemplateUploadSchema.ts

```diff
@@ -16,7 +16,7 @@ export interface UploadByUrlForm extends NamedUpload {
 const NameSchema: ObjectSchemaDefinition<NamedUpload> = {name: string().required('Required').min(3, 'Must have at least 3 characters')};
 const UploadFormSchema = object<UploadForm>({
   ...NameSchema,
-  file: object().nullable() as ObjectSchema<File>
+  file: object() as ObjectSchema<File>
```

Comment:
This is a dangerous suggestion, it breaks the validation message but it was also broken before in a different way. 

it should be this:

```suggestion
  file: object<File>().nullable().required('File is required')
```

Reply:
Committed this change.